.github/scripts/update_disabled_tests.py

```diff
@@ -73,10 +75,41 @@ def write_issues_to_file(issues_json: Dict[Any, Any]) -> None:
         json.dump(issues_json, file, sort_keys=True, indent=2)
 
 
+def condense_disable_issues(disable_issues):
+    disabled_test_from_issues = dict()
+    for item in disable_issues["items"]:
+        title = item["title"]
+        key = "DISABLED "
+        issue_url = item["html_url"]
+        issue_number = issue_url.split("/")[-1]
+        if title.startswith(key):
+            test_name = title[len(key):].strip()
+            body = item["body"]
+            platforms_to_skip = []
+            key = "platforms:"
+            # When the issue has no body, it is assumed that all platforms should skip the test
+            if body is not None:
+                for line in body.splitlines():
+                    line = line.lower()
+                    if line.startswith(key):
+                        pattern = re.compile(r"^\s+|\s*,\s*|\s+$")
```

Comment:
This look a bit weird because a relatively complicated regex is used to just split the string.  I think we could:

* Use split, i.e. `[v.strip() for v in line[len(key):].split(",") if v]`
* or if you prefer the regex, move `pattern = re.compile` out of the for loop.  It seems a waste to re-compile this regex (albeit a tiny waste)